mathesar/filters.py

```diff
@@ -3,6 +3,16 @@
 
 from mathesar.models import Schema, Table, Database
 
+FILTER_OPTIONS_BY_TYPE_NAME = {
+    'Boolean':
+        {
+            'eq': '<boolean>',
+            'ne': '<boolean>',
+            'is_null': None,
```

Comment:
Why is this `None`? Shouldn't it also be a boolean if it's describing the kind of input it takes?